sympy/matrices/expressions/transpose.py

```diff
@@ -37,6 +37,8 @@ def doit(self, **hints):
         if hints.get('deep', True) and isinstance(arg, Basic):
             arg = arg.doit(**hints)
         try:
+            if isinstance(arg, MatrixExpr) and arg.shape == (1, 1):
+                return arg
```

Comment:
It looks like this is related to the predicate `symmetric` in the [assumptions](https://github.com/sympy/sympy/blob/master/sympy/assumptions/ask.py#L733):

       ``Q.symmetric(x)`` is true iff ``x`` is a square matrix and is equal to
        its transpose. Every square diagonal matrix is a symmetric matrix.